src/test/java/io/vertx/test/core/JsonObjectTest.java

```diff
@@ -1120,18 +1120,18 @@ public void testEncodePrettily() throws Exception {
     jsonObject.put("myobj", new JsonObject().put("foo", "bar"));
     jsonObject.put("myarr", new JsonArray().add("foo").add(123));
     String strBytes = Base64.getEncoder().encodeToString(bytes);
-    String expected = "{\n" +
-      "  \"mystr\" : \"foo\",\n" +
-      "  \"myint\" : 123,\n" +
-      "  \"mylong\" : 1234,\n" +
-      "  \"myfloat\" : 1.23,\n" +
-      "  \"mydouble\" : 2.34,\n" +
-      "  \"myboolean\" : true,\n" +
-      "  \"mybinary\" : \"" + strBytes + "\",\n" +
-      "  \"myobj\" : {\n" +
-      "    \"foo\" : \"bar\"\n" +
-      "  },\n" +
-      "  \"myarr\" : [ \"foo\", 123 ]\n" +
+    String expected = "{" + System.getProperty("line.separator") +
```

Comment:
Done.  I changed a few things.  The Windows class contained only one helper method (isWindows).  I renamed that class to Utils and then added LINE_SEPARATOR in there.  Found another place in the code that defined it's own instance, and pointed that at the Utils version too.